### PR TITLE
Da#94 created collection should be automatically visible

### PR DESCRIPTION
**Pull request by AayushTyagi1**
## Describe the problem this PR is solving
When we create a new collection, it does not automatically refresh the UI and update in folder view.

## Short description of the changes
We expect to see the new collection when we create it and we are doing it by calling parent bucket of that collection.
Also the PR contains some error handing where we are handling the issues with DocumentNotFound Error.

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -498,8 +515,9 @@ export function activate(context: vscode.ExtensionContext) {
           name: collectionName,
           scopeName: node.scopeName,
         });
-
+
```
Comment (MikeGoldsmith) [inline src/extension.ts:518]: not: unnecessary white space

File: src/extension.ts [hunk 2]
```diff
@@ -498,8 +515,9 @@ export function activate(context: vscode.ExtensionContext) {
           name: collectionName,
           scopeName: node.scopeName,
         });
-
+        
         clusterConnectionTreeProvider.refresh(node);
+        clusterConnectionTreeProvider.refresh(node.parentNode);
```
Comment (MikeGoldsmith) [inline src/extension.ts:520]: I think you could replace the tree provider refresh call above instead of doing a second call.

**Review by MikeGoldsmith — CHANGES_REQUESTED** [review]
Added feedback. Please could you add description of changes for error handling to the PR description or move to a separate PR?

**Review by deniswsrosa — APPROVED** [review]
Good of now, but we will need to refactor this file pretty soon.

File: src/extension.ts (hunk 2 above)
Comment (AayushTyagi1) [reply src/extension.ts:520]: I test that too but doesn't work that way so need to refresh both.

**Comment by AayushTyagi1** [comment]
> Added feedback. Please could you add description of changes for error handling to the PR description or move to a separate PR?

Actually I added the different commit message for that. I am adding it in description too.

File: src/extension.ts (hunk 2 above)
Comment (MikeGoldsmith) [reply src/extension.ts:520]: The tree provider should be executing in flow-down style. So I would expect refreshing the parent to update all child nodes too. What did you see?

Comment (AayushTyagi1) [reply src/extension.ts:520]: @MikeGoldsmith Same error is seeing here too.
Screenshot 2023-02-20 at 12 59 00 PM

Comment (MikeGoldsmith) [reply src/extension.ts:520]: The looks like it cannot find a local file for a cluster document. Not sure how if that's related or not.

Either way, we can accept as it is and come back as an optimisation if required.

**Review by MikeGoldsmith — APPROVED** [review]

**Pull request merged**